api/users/views.py

```diff
@@ -173,14 +173,16 @@ def get_queryset(self):
         organisation = Organisation.objects.get(id=organisation_pk)
 
         qs = UserPermissionGroup.objects.filter(organisation=organisation)
-        if not self.request.user.has_organisation_permission(
-            organisation, MANAGE_USER_GROUPS
+        if (
+            self.action != "summaries"
```

Comment:
As we talked about offline, the `UserPermissionGroupPermission` needs to be added to handle the `summaries` action so that all users can see the group names.

Reply:
Excellent catch my man! 🙌